rmgpy/data/kinetics/family.py

```diff
@@ -2616,25 +2616,41 @@ def get_labeled_reactants_and_products(self, reactants, products):
         new entities in memory so input molecules `reactants` and `products` won't be affected.
         If RMG cannot find appropriate labels, (None, None) will be returned.
         """
+
```

Comment:
Was this addition intentional?

Reply:
I don't think so. I was moving code around before settling on the final version of the fix.